app/assets/stylesheets/browse.scss

```diff
@@ -100,6 +100,7 @@
   display: flex;
   justify-content: space-between;
   position: relative;
+  box-shadow: 2px 30px 60px rgba(0, 0, 0, 0.05);
```

Comment:
I don't see the point of adding this shadow, it was clear and going well with the whole style, do you have any justification for doing this?

Reply:
The shadow was to have a better distinction between the blocks(cards), and in Mac screens, it was too bright.